src/IO/Exporter/PointwiseInterpolator.cpp

```diff
@@ -152,10 +152,16 @@ void interpolate_to_points_impl(
 template <size_t Dim>
 void interpolate_to_point_impl(
     const gsl::not_null<std::vector<double>*> result,
-    const tnsr::I<DataVector, Dim, Frame::ElementLogical>& x_element_logical,
+    const tnsr::I<double, Dim, Frame::ElementLogical>& x_element_logical,
     const Mesh<Dim>& mesh, const size_t offset, const size_t length,
     const std::vector<DataVector>& tensor_data) {
-  const intrp::Irregular<Dim> interpolant(mesh, x_element_logical);
+  // Copy into DataVector can be elided by supporting interpolation of double in
+  // intrp::Irregular
+  tnsr::I<DataVector, Dim, Frame::ElementLogical> x_element_logical_dv(1_st);
+  for (size_t d = 0; d < Dim; ++d) {
+    x_element_logical_dv.get(d) = x_element_logical.get(d);
+  }
```

Comment:
Just make this change. Dealing with a lot of allocations was a pain in spec and caused a lot of inefficiencies. Comments like this get lost easily.

Reply:
Done. Also added a commit to interpolate single-precision data.